src/util.rs

```diff
@@ -93,6 +93,15 @@ pub(crate) fn base_name(path: &PathBuf) -> &str {
         .expect("must be valid utf-8")
 }
 
+/// Returns `true` if output should be printed (i.e. verbose output is set).
+pub(crate) fn is_verbose(verbosity: &Option<Verbosity>) -> bool {
```

Comment:
If we make `Verbosity` a three variant enum e.g. `Default, Quiet, Verbose`, we can replace `Option<Verbosity>` and make `is_verbose` a method.